### PR TITLE
Update for runner custom names

### PR DESCRIPTION
**Pull request by mpas**
## Description
Add option to define name of runner instance and security groups

## Migrations required
NO

## Verification
public example

### CODE REVIEW

**Comment by npalm** [comment]
@mpas Thanks a lot for proposing a PR. Please can you have a quick look at the PR template?

File: main.tf [hunk 1]
```diff
@@ -9,6 +9,18 @@ locals {
...
+
+    // Define key for runner token for SSM
+    secure_parameter_store_runner_token_key = "${var.environment}-${var.secure_parameter_store_runner_token_key}"
+
+    // custom names for instances and security groups
+    name_runner_instance = "${var.overrides["name_runner_agent_instance"] == "" ? local.tags["Name"] : var.overrides["name_runner_agent_instance"]}"
+    name_sg              = "${var.overrides["name_sg"] == "" ? local.tags["Name"] : var.overrides["name_sg"]}"
 }
 
 resource "aws_security_group" "runner" {
```
Comment (npalm) [inline main.tf]: I have just solved #62 which cause a change set every time running an apply. Therefore an extra security group  is introduced. Please can you sync with develop and updated the new security group with this change as well.

Comment (mpas) [reply main.tf]: Can you check if all is ok now?

**Comment by npalm** [comment]
@mpas Thanks for the update will check your PR.

**Pull request merged**